src/traj.c

```diff
@@ -27,6 +29,7 @@ static DebugStrings_t TRAJInfoDescription = 		{"Trajectory info",	"",	&printU8};
  *		EMSGSIZE	if trajectory name is too long
  */
 ssize_t encodeTRAJMessageHeader(
+	const uint32_t receiverID,  const uint8_t messageCounter,
```

Comment:
Small detail but move messageCounter to its own row to follow the structure of one parameter per row.